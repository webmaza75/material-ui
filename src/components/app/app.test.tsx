import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import App from './app';

describe(`App`, () => {
  it(`App renders correctly`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const result = renderer
      .render(<App />);

    expect(result).toMatchSnapshot();
  });
});
