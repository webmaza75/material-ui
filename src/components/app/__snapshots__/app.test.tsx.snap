// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App renders correctly 1`] = `
<WithStyles(CssBaseline)>
  <Switch>
    <div
      style={
        Object {
          "flexGrow": 1,
        }
      }
    >
      <Context.Provider
        value={
          Object {
            "addTask": [Function],
            "deleteTasks": [Function],
            "taskList": Array [
              Object {
                "id": 1,
                "repeat": Array [
                  6,
                  0,
                  1,
                  2,
                  3,
                  4,
                  5,
                ],
                "reportTime": "Wed Jul 17 2019 10:00:13 GMT+0300",
                "timeZone": "Asia/Aqtu UTC+5:00",
                "title": "Auto Orica USA report setting",
                "type": "Payload Monitoring Report",
              },
              Object {
                "id": 2,
                "repeat": Array [
                  0,
                  6,
                ],
                "reportTime": "Wed Jul 17 2019 09:00:13 GMT+0300",
                "timeZone": "Asia/Aqtu UTC+5:00",
                "title": "Motion Metrics International",
                "type": "Payload Monitoring Report",
              },
              Object {
                "id": 3,
                "repeat": Array [
                  1,
                  2,
                  3,
                  4,
                ],
                "reportTime": "Wed Jul 17 2019 15:00:13 GMT+0300",
                "timeZone": "America/Los_Angeles UTC+8:00",
                "title": "MPM auto weekly report task",
                "type": "Fragmentation Report",
              },
              Object {
                "id": 4,
                "repeat": Array [
                  4,
                ],
                "reportTime": "Wed Jul 17 2019 09:30:13 GMT+0300",
                "timeZone": "Africa/Johannesburg UTC+2:00",
                "title": "Orica USA auto report",
                "type": "Tooth Detection Report",
              },
              Object {
                "id": 5,
                "repeat": Array [
                  1,
                  2,
                  4,
                  5,
                  3,
                ],
                "reportTime": "Wed Jul 17 2019 09:30:13 GMT+0300",
                "timeZone": "Africa/Johannesburg UTC+2:00",
                "title": "Orica USA auto report",
                "type": "Tooth Detection Report",
              },
            ],
            "undoDeleteTasks": [Function],
          }
        }
      >
        <Navbar />
        <Route
          component={[Function]}
          exact={true}
          path="/"
        />
        <Route
          component={[Function]}
          path="/add/"
        />
      </Context.Provider>
    </div>
  </Switch>
</WithStyles(CssBaseline)>
`;
